swupd/swupd_test.go

```diff
@@ -196,3 +198,44 @@ func TestProcess(t *testing.T) {
 		t.Fatal("Message processed incorrectly. Expected: success, Actual:", mp.output)
 	}
 }
+
+func TestOffline(t *testing.T) {
+	options := args.Args{
+		SwupdVersion: "latest",
+	}
+
+	if err := utils.ParseOSClearVersion(); err != nil {
+		t.Fatalf("Could not parse OS version")
+	}
+
+	if OfflineIsUsable("latest", options) {
+		t.Fatalf("Offline Content should not be usable to no content dir")
+	}
+
+	if !utils.IsRoot() {
```

Comment:
Can we move this to top of the test?

Reply:
It drops the coverage for non-root testing by almost 1%, but moved.

Reply:
moved back because of the coverage numbers.